build/pmm-client-docker/Dockerfile-alpine

```diff
@@ -0,0 +1,31 @@
+FROM alpine:latest
+
+ENV MONIT_VERSION=5.25.2
+
+# Compile and install monit
+RUN apk add --update monit bash
+
+COPY pmm-client.tar.gz /tmp/pmm-client.tar.gz
+RUN tar -zxpf /tmp/pmm-client.tar.gz -C /tmp \
+    && cd /tmp/pmm-client-* \
+    && bash ./install
+
+
+FROM alpine:latest
+RUN apk add --update --no-cache bash curl
+
+# golang binaries typically have only one dynamic library dependency - /lib64/ld-linux-x86-64.so.2
+# but Alpine Linux uses a smaller realization
+RUN mkdir /lib64 && ln -s /lib/libc.musl-x86_64.so.1 /lib64/ld-linux-x86-64.so.2
```

Comment:
No-no-no. `ld-linux` is not a libc and not even a real library. pmm-client should not depend on any libc, and using musl can cause problems.
Let's figure out why is this needed at all. Wild guess: we are not compiling it with `CGO_ENABLED=0` environment variable.

Reply:
problem gone with CGO_ENABLED=0